docs/ops/generation/Multinomial_13.md

```diff
@@ -62,7 +62,7 @@ Example 3 - 1D tensor, without replacement
 * ``output_type``
 
   * **Description**: the type of the output. Determines generation algorithm and affects resulting values. Output numbers generated for different values of *output_type* may not be equal.
-  * **Range of values**: "i32", "i64", "f16", "bf16", "f32", "f64".
+  * **Range of values**: "i4", "i8", "i16", "i32", "i64", "f32", "u4", "u8", "u16", u32", "u64".
```

Comment:
Just curious. Will be there dedicated implementation for `binary` precision `i4`, `u4` in reference implementation?

Reply:
Afaik no, the conversion is simply applied to the output indices at the last step, so I believe it should work just fine (as long as no overflow is present)

Reply:
Also as index value is always an integer, it seems like supporting "i32" and "i64" as usual for such `output_type` attribute seems to be enough. Example in [Unique_10.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Unique_10.md) spec.
Then conversion of this values to any other type is possible as additional step.
But it can be easier to clarify that a little bit later, based on the tests and implementation.

Reply:
Removed other values except for i32 and i64, as it also helps with Paweł's comment regarding overload of template implementations. Updated the spec!